app/src/main/res/layout/item_notification_filter.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:layout_height="@dimen/list_item_default_height"
+    tools:layout_width="match_parent"
+    tools:parentTag="android.widget.LinearLayout">
+
+    <TextView
+        android:id="@+id/notification_filter_language_code"
+        style="@style/TextViewCentered"
+        android:layout_width="20dp"
+        android:layout_height="20dp"
+        android:layout_gravity="center_vertical"
+        android:layout_marginStart="24dp"
+        android:background="@drawable/lang_button_shape_border"
+        android:fontFamily="sans-serif-medium"
+        android:textAllCaps="true"
+        android:textSize="10sp"
+        android:textColor="@color/base30" />
```

Comment:
This only works for two-letters language codes. e.g. `zh-hant` language code text will be cut.